content/logs/index.md

```diff
@@ -113,7 +116,7 @@ logs:
 Set `type` to **tcp** or **udp** depending of your protocol then specify the `port` of your incomming connection.
 
 Example: 
-If your PHP application doesn't log into a file but forwards its logs locally to TCP port 10518, create a `php.yaml` file as such in order to listen to this port and forward those logs to your datadog application.:
+If your PHP application doesn't log into a file but forwards its logs locally to TCP port 10518, create a `php.yaml` file like so in order to listen to this port and forward those logs to your datadog application:
```

Comment:
"If your PHP application does not log to a file, but instead forwards its logs via TCP, you will need to  create a configuration file that specifies the port to receive as in the example below":

<insert example>

How did we come up with 10518 here?

Reply:
We just took a common port for local log forwarding